src/index.tsx

```diff
@@ -7,6 +7,24 @@ import { Router } from './Router';
 import './Extensions';
 import GlobalErrorBoundaryState from './Utilities/ErrorBoundaries/GlobalErrorBoundary';
 import { VersionCheck } from './Utilities/VersionCheck';
+import axios from 'axios';
+
+interface RederlyConfig {
+    paymentURL: string;
+}
+
+declare global {
+    interface Window {
+        rederlyConfig?: Promise<RederlyConfig>;
+    }
+}
+
+const getConfig = async () => {
+    window.rederlyConfig = axios.get<RederlyConfig>(`/config.json?cache_bust=${new Date().getTime()}`).
+        then(resp => resp.data);
```

Comment:
.catch(err => {
    logger.error('Failed to fetch frontend configuration', e);
    return null;
});